std_msgs/Header header

string token
string instance_token
string sample_token

Category category
Attribute[] attributes
string visibility_token

geometry_msgs/Point translation
geometry_msgs/Quaternion rotation
geometry_msgs/Vector3 size # length, width, height

int32 num_lidar_pts
int32 num_radar_pts
